src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -850,6 +881,15 @@ private function registerRouterConfiguration(array $config, ContainerBuilder $co
 
         $loader->load('routing.xml');
 
+        $this->addClassesToPreload([
+            'Symfony\Bundle\FrameworkBundle\Routing\RedirectableCompiledUrlMatcher',
+            'Symfony\Component\Routing\Annotation\Route',
```

Comment:
not needed IMO. Annotations are used only on cache warmup

Reply:
the annotation class loader still instantiates them from the cache